isis/tests/FuntionalTestsFindImageOverlaps.cpp

```diff
@@ -0,0 +1,53 @@
+#include "findimageoverlaps.h"
+#include "Fixtures.h"
+#include "TestUtilities.h"
+#include "IException.h"
+#include "FileList.h"
+#include "ImagePolygon.h"
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/findimageoverlaps.xml").expanded();
+
+TEST_F(ThreeImageNetwork, FunctionalTestFindImageOverlapsNoOverlap) {
+  ImagePolygon fp1;
+  fp1.Create(*cube1);
+  cube1->write(fp1);
+  
+  Cube newCube2;
+  json newIsd2; 
+  std::ifstream i(isdPath2->expanded().toStdString());
+  i >> newIsd2; 
+
+  newIsd2["InstrumentPosition"]["Positions"] = {{1,1,1}, {2,2,2}, {3,3,3}};
+  newCube2.fromIsd(tempDir.path()+"/new2.cub", *cube2->label(), newIsd2, "rw"); 
+
```

Comment:
I am not a fan of this approach to modifying footprints for images. Preferably, I'd be able to create a blob object with a WKT string and add it directly to the Cube but the API doesn't seem to support this?